modules/system/console/OctoberUtil.php

```diff
@@ -328,4 +331,27 @@ protected function utilGitPull()
         }
     }
 
+    /**
```

Comment:
remove this docblock

Reply:
resolved in https://github.com/octobercms/october/pull/3946/commits/0680384f9a121b4e56a14d322f1f64723de238d2